Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/Extensions/ApiGatewayResponseTestCases.cs

```diff
@@ -0,0 +1,367 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text;
+using System.Text.Json;
+using Amazon.Lambda.APIGatewayEvents;
+
+namespace Amazon.Lambda.TestTool.UnitTests.Extensions;
+
+public static class ApiGatewayResponseTestCases
+{
+    public static IEnumerable<object[]> V1TestCases()
```

Comment:
these are all the same. the only minor change that i did (as mentioned in the pr description) is remove the unneeded "Assertions" functions